packages/vue3-vite/src/generators/vue3-vite/files/vite.config.ts.template

```diff
@@ -27,4 +27,9 @@ export default defineConfig({
       dirs: ['src/app/components'],
     }),
   ],
+  <% if (testFramework === 'vitest') { -%>
+    test: {
+      environment: 'happy-dom',
+    }
+  <% } %>
```

Comment:
`test` option is required in `vite.config.ts` if we want to use Vitest to test components.